app/Http/Controllers/UserController.php

```diff
@@ -122,6 +124,7 @@ public function deleteAvatar(Request $request): JsonResponse
         $this->authorize('updateAvatar', $user);
 
         removeAvatar($user->username);
+        $this->purgeUserAvatarFromCloudflare($user);
```

Comment:
`removeAvatar` is also called when banning a user and clearing account data. Should this logic be moved into `media.php` and called from `removeAvatar` and `UploadAvatar`?